bundles/org.aposin.mergeprocessor/src/org/aposin/mergeprocessor/renaming/RenameQueryExecutor.java

```diff
@@ -66,14 +66,14 @@
 		Boolean result = results.get(mergeUnit);
 		if (result != null) {
 			// result available
-			return new DoneFuture<Boolean>(result);
+			return ConcurrentUtils.constantFuture(result);
```

Comment:
Replaced usage of `org.h2.util.DoneFuture` to become more independent to H2.

Reply:
Nice!